kolibri/core/assets/src/api_resource.js

```diff
@@ -254,7 +411,6 @@ class Resource {
   getModel(id, data = {}) {
     let model;
     if (!this.models[id]) {
-      Object.assign(data, { [this.idKey]: id });
```

Comment:
so in the case of calling getModel does not find the model, should I call fetch on the server or createModel?